packages/select/src/Select/components/Trigger/SingleTriggerButton/SingleTriggerButton.tsx

```diff
@@ -0,0 +1,34 @@
+import React from "react"
+import classnames from "classnames"
+import { RootProps } from "../../Root"
+import { TriggerButtonBase } from "../TriggerButtonBase"
+import styles from "./SingleTriggerButton.module.scss"
+
+export type SingleTriggerButtonProps = {
+  placeholder: string
+  selectedOptionLabel: RootProps["selectedKey"]
+  classNameOverride?: string // TODO: migrate it to use OverrideClassName<T> and omit the props controlled by React-Aria
+}
+
+export const SingleTriggerButton: React.VFC<SingleTriggerButtonProps> = ({
+  selectedOptionLabel,
+  classNameOverride,
+  placeholder,
+}) => (
+  <TriggerButtonBase
+    classNameOverride={classnames([
+      selectedOptionLabel === null && styles.placeholder,
+      classNameOverride,
+    ])}
+  >
+    {
+      <span>
+        {selectedOptionLabel != null
+          ? `${selectedOptionLabel}`
+          : `${placeholder}`}
+      </span>
```

Comment:
```suggestion
<span>{`${selectedOptionLabel ?? placeholder}`}</span>
```
Nice little trick https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing

Reply:
You might not need the `${}` too :)